internal/keys/keys.go

```diff
@@ -77,8 +77,11 @@ func (k *KeyResult) String() string {
 	if k.accountKey != nil {
 		_, _ = fmt.Fprintf(writer, "Signature algorithm \t %s\n", k.accountKey.SigAlgo)
 		_, _ = fmt.Fprintf(writer, "Hash algorithm \t %s\n", k.accountKey.HashAlgo)
-		_, _ = fmt.Fprintf(writer, "Weight \t %d\n", k.accountKey.Weight)
 		_, _ = fmt.Fprintf(writer, "Revoked \t %t\n", k.accountKey.Revoked)
+
+		if k.accountKey.Weight >= 0 { // dont show empty value
```

Comment:
Won't `k.accountKey` be `0` when empty?

Reply:
Yes, this was put in place because when you do PEM decoding the result is returned as`AccountKey` but I don't want to display weight in the result since it is not known and I set it as -1 which then gets omitted in the display. If you have a better idea please let me know as I don't really like it, but it does the job without creating new structs. I do want to display keys with weight 0 since I believe this is valid.

Reply:
Ah I see, didn't realize it was being set to -1.